tools/explorer/tt_adapter/src/tt_adapter/runner.py

```diff
@@ -131,8 +135,31 @@ def run(
             "DEVICE FW DURATION [ns]",
             "CORE COUNT",
             "OUTPUT_0_MEMORY",
+            "LOC",
         ]
         perf = perf[columns]
-        print(perf)
 
-        print("Total device duration: ", perf["DEVICE FW DURATION [ns]"].sum(), "ns")
```

Comment:
Please leave these in for now.

Reply:
It will be handled differently when we implement an actual logging solution tho.

Reply:
Alright I'll throw these back in then.